parachain/runtime/testnet-kintsugi/src/lib.rs

```diff
@@ -668,6 +672,9 @@ impl orml_tokens::Config for Runtime {
     type WeightInfo = ();
     type ExistentialDeposits = ExistentialDeposits;
     type OnDust = orml_tokens::TransferDust<Runtime, FeeAccount>;
+    type OnSlash = ();
+    type OnDeposit = ();
+    type OnTransfer = ();
```

Comment:
Do these not need to be set?

Reply:
yes they do... which is exactly why I need to add more integration tests. Seems like I had only added these to the mock runtime